Cargo.toml

```diff
@@ -34,6 +34,7 @@ digest = { version = "0.9", features = ["dev"] }
 default = ["aead", "alloc", "digest", "signature"]
 alloc = ["aead/alloc"]
 signature = ["ecdsa", "ed25519", "p256", "p384"]
+std = ["digest/std", "ecdsa/std", "ed25519/std", "p256/std", "p384/std"]
```

Comment:
`"p256/std", "p384/std"` is probably superfluous, since they use `signature::Error`, and all that really matters is receiving a `std::error::Error` impl for that

Reply:
Dropped those two (thanks for pointing that out, I am not especially familiar with those details)